packages/sdk/src/index.ts

```diff
@@ -60,13 +61,33 @@ export default class SDK {
     opts: InitOptions = {
       logEndpointInitTime: true,
       ipfsClientUrl: "https://ipfs.zeitgeist.pm",
+      initialConnectionTries: 5,
     }
   ): Promise<SDK> {
     try {
       const start = Date.now();
-      const api = await SDK.promiseWithTimeout(
-        10000,
-        initApi(endpoint),
+
+      const api = await SDK.promiseWithTimeout<ApiPromise>(
+        15000,
```

Comment:
Could we increase this a bit more? Polkadot js use 60 seconds by default so I think something similar would be good

Reply:
Is the timeout here `15000` has a different function? Looks like we can scrap `promiseWithTimeout` thing.

https://github.com/zeitgeistpm/tools/blob/2798a9ff1604568f0dc53f65fa47731d55d1e9ba/packages/sdk/src/util/polkadot.ts#L144

Reply:
> Does the timeout `15000` have a different usage here? Looks like we can scrap `promiseWithTimeout` thing.
> 
> https://github.com/zeitgeistpm/tools/blob/2798a9ff1604568f0dc53f65fa47731d55d1e9ba/packages/sdk/src/util/polkadot.ts#L144

No I thought that too, but it is just per connection attempt. It still retries indefinitely even if it is set.

Reply:
> Could we increase this a bit more? Polkadot js use 60 seconds by default so I think something similar would be good

Yes makes sense, since it retries 5 times only fast failing connections due to not found or invalid url will fail quick anyway, and lets the user wait for slow endpoints.